lib/state/ui/actions.ts

```diff
@@ -91,6 +91,20 @@ export const showDialog: A.ActionCreator<A.ShowDialog> = (
   dialog,
 });
 
+export const showTokenLoginLogout: A.ActionCreator<A.ShowTokenLoginLogout> = (
+  email: string
+) => ({
+  type: 'SHOW_TOKEN_LOGIN_LOGOUT',
```

Comment:
this action name is pretty confusing to me. I don't know if there are other options but I'm not sure what it's purpose is, what interaction it's matching.

Reply:
I changed this component to `AlternateLoginPrompt` and the action to`SHOW_ALTERNATE_LOGIN_PROMPT`